test/tests/server.js

```diff
@@ -245,6 +246,28 @@ describe('Server', () => {
     });
   });
 
+  describe('MultiCompiler: One `publicPath`', () => {
+    before((done) => {
+      app = express();
+      const compiler = webpack(webpackClientServerConfig);
+      instance = middleware(compiler, {
+        stats: 'errors-only',
+        logLevel: 'silent'
+      });
+      app.use(instance);
+      listen = listenShorthand(done);
+    });
+    after(close);
+
+    it('request to bundle file', (done) => {
+      request(app).get('/static/foo.js').expect(200, done);
+    });
+
+    it('request to non-public path', (done) => {
```

Comment:
should also test for a non-existent resource on a valid public path for thoroughness. eg. `/static/foo.js`